documentation/API_documentation/NumberVerification_verify_User_Story.md

```diff
@@ -0,0 +1,10 @@
+# Number Verification Verify API User Story
+
+| **Item** | **Details** |
+| ---- | ------- |
+| ***Summary*** | As an enterprise application developer, I want to verify the phone number associated with the line from which the call was made, so that I can ensure that I avoid identity theft fraud. |
+| ***Roles, Actors and Scope*** | **Roles:** Customer:User, Customer:BusinessManager, Customer:Administrator<br> **Actors:** Application service providers, hyperscalers, application developers, end users. <br> **Scope:**  <br> - Verifies if the specified phone number (plain text or hashed format) matches the one that the user is currently using. |
+| ***Pre-conditions*** |The preconditions are listed below:<br><ol><li>The Customer:BusinessManager and Customer:Administrator have been onboarded to the CSP's API platform.</li><li>The Customer:BusinessManager has successfully subscribed to the Number Verification product from the product catalog.</li><li>The Customer:Administrator has onboarded the Customer:User to the platform.</li><li>The Customer:user performs an authorization request to CSP</li><li>The means to get the access token are known to the Customer:User to ensure secure access of the API.|
+| ***Activities/Steps*** | **Starts when:** The customer application makes a POST verify via the number verification API providing in the request the phone number keyed by the user on the application. This input could be hashed or plain.<br>**Ends when:** The Number verification server answers if the phone number provided corresponds to the one of the line from which the request was triggered. |

```

Comment:
the phone number keyed by the user on the application
--> the phone number provided by the end user in the application